README.md

```diff
@@ -1,33 +1,61 @@
-Right now that means the geometries are hard to use, but at least parsing is fast.
+properties in the columns individually. The geometry and non-scalar properties are kept as GeometryBasics geometries and NamedTuple respectively, and both of them are wrapped in a `Feature`, while a StructArray of Features represents a `FeatureCollection`. The StructArrays approach is inspired by GeometryBasics.
+For a faster/lower level interface, the geojson can be can also be parsed as a JSON3 dictionary.
 
-Going forward, it would be nice to try developing a GeoTables.jl, similarly to Tables.jl, but with special support
-for a geometry column, that supports a diverse set of geometries, such as those of [LibGEOS](https://github.com/JuliaGeo/LibGEOS.jl), [Shapefile](https://github.com/JuliaGeo/Shapefile.jl), [ArchGDAL.jl](https://github.com/yeesian/ArchGDAL.jl/), [GeometryBasics](https://github.com/SimonDanisch/GeometryBasics.jl) and of course this package.
+Going forward, it would be nice to try developing a GeoTables.jl, similarly to Tables.jl, but with special support for a geometry column, that supports a diverse set of geometries, such as those of [LibGEOS](https://github.com/JuliaGeo/LibGEOS.jl), [Shapefile](https://github.com/JuliaGeo/Shapefile.jl), 
+[ArchGDAL.jl](https://github.com/yeesian/ArchGDAL.jl/), [GeometryBasics](https://github.com/SimonDanisch/GeometryBasics.jl) and of course this package.
 
 It would also be good to explore integrating this code into [GeoJSON.jl](https://github.com/JuliaGeo/GeoJSON.jl) and
 archiving this package. See [GeoJSON.jl#23](https://github.com/JuliaGeo/GeoJSON.jl/pull/23) for discussion.
 
 ## Usage
 
 ```julia
-julia> using GeoJSONTables, DataFrames
+julia> using GeoJSONTables, DataFrames, GeometryBasics
 
 julia> jsonbytes = read("path/to/a.geojson");
 
+#= 
+   A true flag can be passed in the read function 
+   to read the jsonbytes as a JSON3 dict 
+   the default, false, will read it into a StructArray 
+=#
 julia> fc = GeoJSONTables.read(jsonbytes)
 FeatureCollection with 171 Features
 
+julia> fc_json3 = GeoJSONTables.read(jsonbytes, true)
+JSON3.Object{Array{UInt8,1},Array{UInt64,1}} with 4 entries:
+    :type     => "FeatureCollection"
+    :name     => "a"
+    :crs      => {…
+    :features => JSON3.Object[{…
+
 julia> first(fc)
-Feature with geometry type Polygon and properties Symbol[:geometry, :timestamp, :version, :changeset, :user, :uid, :area, :highway, :type, :id]
+Feature with geometry type Polygon and properties (:geometry, :timestamp, :version, :changeset, :user, :uid, :area, :highway, :type, :id)
 
 # use the Tables interface to convert the format, extract data, or iterate over the rows
 julia> df = DataFrame(fc)
+
+# GeometryBasics geometries can be passed along with metadata, into a Feature
+julia> f = GeoJSONTables.geo2feat(Point(1.0, 2.0), city = "Mumbai", rainfall = 1010)
```

Comment:
Yeah. https://github.com/visr/GeoJSONTables.jl/blob/meta/src/GeoJSONTables.jl#L118-L126

Reply:
So `geo2feat()` will handle it, even the `missing` cases nicely.

Reply:
I think the link is outdated now, not sure what lines you were referring to. It's good to press y in github before copying the link, that will put the commit hash in there such that the lines will stay the ones you mean.

But I don't understand your last comment. So `geo2feat()` does things that `Feature()` doesn't related to `missing`? What is that exactly? And can we put that behavior in the `Feature` constructor as well?

Reply:
Okay. Thanks!
So the link was referring to `geo2feat` methods https://github.com/visr/GeoJSONTables.jl/blob/a8fc7bbfa61045dd1103ff9c5fd69a712b86a2fe/src/GeoJSONTables.jl#L105-L113

So what we want is, like you said earlier that even something like a `Point(1, 2)` is a valid geojson format and we should accept it as well. And I also wanted to be consistent with the currently existing `GeoJSON.jl` package. [This](https://github.com/visr/GeoJSONTables.jl/issues/5#issuecomment-659621092 ) comment includes the details. 

We can also use the existing `Feature` constructor as well. But I was concerned that it might run into conflicts with the actual usage of Feature [here](https://github.com/visr/GeoJSONTables.jl/blob/a8fc7bbfa61045dd1103ff9c5fd69a712b86a2fe/src/GeoJSONTables.jl#L34-L41) for which the [constructor](https://github.com/visr/GeoJSONTables.jl/blob/a8fc7bbfa61045dd1103ff9c5fd69a712b86a2fe/src/GeoJSONTables.jl#L12) already exists.

Reply:
> for which the constructor already exists

Luckily we have multiple dispatch

> Point(1, 2) is a valid geojson format and we should accept it as well

True, but that should just be a point and not a Feature. I don't see what `geo2feat` does in general, other than acting as a `Feature` constructor.

Reply:
Ah sure. I'll try it out then.